pom.xml

```diff
@@ -204,5 +218,24 @@
 			</properties>
 		</profile>
 
+		<profile>
+			<id>Java 17</id>
+			<activation>
+				<jdk>17</jdk>
+			</activation>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<version>3.2.2</version>
```

Comment:
Can't we use the same version as Java 8?

Reply:
we use the same version as used in java 8

Reply:
Just so you know, you don't need to state the version here explicitly.  You could remove the version here.